src/vw/src/main/scala/VectorUDTUtil.scala

```diff
@@ -0,0 +1,11 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package org.apache.spark.ml
+
+import org.apache.spark.ml.linalg.VectorUDT
```

Comment:
I think you are supposed to use:

import org.apache.spark.ml.linalg.SQLDataTypes.VectorType

for non spark applications

Reply:
thanks!